packages/replay-next/src/suspense/EventsCache.ts

```diff
@@ -166,10 +166,12 @@ const CountEvents: Partial<Record<RecordingTarget, CountEventsFunction>> = {
     const chromiumEventCategoriesByEventDefault = makeChromiumEventCategoriesByEventDefault();
     const chromiumEventCategoriesByEventAndTarget = makeChromiumEventCategoriesByEventAndTarget();
     const normalizedEventCounts: {
-      [key: string]: {
-        count: number;
-        rawEventTypes: string[];
-      };
+      [key: string]:
+        | {
+            count: number;
+            rawEventTypes: string[];
+          }
+        | undefined;
```

Comment:
cc @markerikson @hbenl I would have expected TypeScript to catch this as potentially `undefined`, but [it doesn't](https://www.typescriptlang.org/play?#code/MYewdgzgLgBAtgQwA4C4YG8BQMYG0DWApgJ5rQBOAlmAOYC6aWOOoArmFGmK3AEaHkA3NhgBfYaJgBeDOMyZQkWADcEAG1aFp8ZLgBEAMxAg9dYQHpzMAAJQIAWkIAPJIWBRH5ciHIxVGrUoIGCQQKEIOSnU1Yhh2ABNCA2pCeMx-TQA6Ng4AalzBIA).

The only way I know to fix this is to [mark the value as possibly undefined](https://www.typescriptlang.org/play?ssl=5&ssc=3&pln=1&pc=1#code/JYOwLgpgTgZghgYwgAgLJwA4ElzXk5AbwChlkBtAawgE8AuZAZzClAHMBdBksshAewCu4BiEEBbAEbQA3KWQBfZAB9kwgCYQYoCOrkK5xASGbJxmBumy5YiFAF4iB4kf4mwyAG5wANoIdmmOQARDD8-MEccgD00cgAAmCMALQQAB4YEAhgqVBQ-FBevv7IwIzIGPyQ4MC+PjRqIJraILrE3n4QAHQCwmAA1P0yQA).